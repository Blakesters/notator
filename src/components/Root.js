import React, { Component } from 'react';
import {Instructions} from './Instructions';
import AudioButton from './AudioButton';
import SheetMusic from './SheetMusic';
import SignupForm from './SignupForm';
import './Root.css';

class Root extends Component {
  render() {
    return (
      <div className="Root">
        <header>
          <h1 className="title">Sing or Play Into the Microphone and Press Keys to Create Notation</h1>
          <h3 className="range">The range is F1 â€“ C7</h3>
        </header>
        <AudioButton />
        <SignupForm />
        <Instructions />
        <SheetMusic />
      </div>
    );
  }
}

export default Root;
