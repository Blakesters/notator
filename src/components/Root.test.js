import React from 'react';
import Root from './Root';
import {shallow} from 'enzyme';

describe('Root', () => {
  it('should render without crashing', () => {
    const component = shallow(<Root />);
    expect(component).toMatchSnapshot();
  });
});