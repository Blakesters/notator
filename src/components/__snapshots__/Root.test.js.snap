// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root should render without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Root />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header>
          <h1
                    className="title"
          >
                    Sing or Play Into the Microphone and Press Keys to Create Notation
          </h1>
          <h3
                    className="range"
          >
                    The range is F1 – C7
          </h3>
</header>,
        <Connect(AudioButton) />,
        <ReduxForm />,
        <Instructions />,
        <Connect(SheetMusic) />,
      ],
      "className": "Root",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="title"
>
              Sing or Play Into the Microphone and Press Keys to Create Notation
</h1>,
            <h3
              className="range"
>
              The range is F1 – C7
</h3>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Sing or Play Into the Microphone and Press Keys to Create Notation",
              "className": "title",
            },
            "ref": null,
            "rendered": "Sing or Play Into the Microphone and Press Keys to Create Notation",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "The range is F1 – C7",
              "className": "range",
            },
            "ref": null,
            "rendered": "The range is F1 – C7",
            "type": "h3",
          },
        ],
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header>
            <h1
                        className="title"
            >
                        Sing or Play Into the Microphone and Press Keys to Create Notation
            </h1>
            <h3
                        className="range"
            >
                        The range is F1 – C7
            </h3>
</header>,
          <Connect(AudioButton) />,
          <ReduxForm />,
          <Instructions />,
          <Connect(SheetMusic) />,
        ],
        "className": "Root",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className="title"
>
                Sing or Play Into the Microphone and Press Keys to Create Notation
</h1>,
              <h3
                className="range"
>
                The range is F1 – C7
</h3>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Sing or Play Into the Microphone and Press Keys to Create Notation",
                "className": "title",
              },
              "ref": null,
              "rendered": "Sing or Play Into the Microphone and Press Keys to Create Notation",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "The range is F1 – C7",
                "className": "range",
              },
              "ref": null,
              "rendered": "The range is F1 – C7",
              "type": "h3",
            },
          ],
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
